ansible/test/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Bring up docker containers
+  hosts: localhost
+  gather_facts: false
+  vars:
+    inventory:
+      - name: smtp-outbound
+        image: "geerlingguy/docker-centos7-ansible"
+  roles:
+    - role: provision_docker
+      provision_docker_inventory: "{{ inventory }}"
+      provision_docker_privileged: True
+
+- name: Setup docker pre-reqs
+  hosts: smtp-outbound
+  become: true
+  tasks:
+    - name: Install openssh
+      package:
+        name: openssh-server
```

Comment:
Why does the local docker need openssh, if it has any affect on the SMTP setup then its probably required within the role itself.
Dont forget that even if the server has it, we can still include it to confirm.

Reply:
Removed as not required and should be covered as base AMI.